custom_components/luxtronik/const.py

```diff
@@ -371,6 +372,17 @@ class LuxMkTypes(Enum):
         entity_category=EntityCategory.DIAGNOSTIC,
         sensor_key="calculations.ID_WEB_WMZ_Seit",
     ),
+        LuxtronikSensorEntityDescription(
+        key="current_heat_output",
+        # name=f"{text_heat_amount_counter}",
```

Comment:
`name=f"{text_current_heat_output}"`, but it's commented out, doe we still have to add this for new sensors?

Reply:
doesn't work and is commented out for other sensors as well.

Reply:
OK, but the commented out text is wrong.